src/dist/replication/lib/replica_stub.cpp

```diff
@@ -1712,6 +1740,7 @@ void replica_stub::on_disk_stat()
     _counter_replicas_garbage_replica_dir_count->set(garbage_replica_dir_count);
 
     _fs_manager.update_disk_stat();
+    update_disk_holding_replicas();
```

Comment:
`on_disk_stat` is called by timer, time interval is `options.disk_stat_interval_seconds`, whose default value is 600 seconds. Is this time interval suitable for updating disk primary and secondary stat?

Reply:
I don't think it makes much difference, because it won't report falcon for "health monitor", but only for `shell` query some time.